udapi/block/write/asciitree.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+
+import logging
+import codecs
+import sys
+
+from udapi.core.block import Block
+
+
+def get_number_of_antecedents(node):
```

Comment:
See https://github.com/udapi/udapi-perl/blob/master/lib/Udapi/Block/Write/TextModeTrees.pm
- [ ] add here a link to the Perl implementation as a TODO (parameters, colors, nonprojectivities etc.)
- [ ] I think we should call this block the same in Perl, Java and Python: I think both TextModeTrees and AsciiTrees is ok, with a slight preference to the former.